src/validation_workflow/validation.py

```diff
@@ -51,6 +53,27 @@ def check_for_security_plugin(self, work_dir: str) -> bool:
         path = os.path.exists(os.path.join(work_dir, "plugins", "opensearch-security"))
         return path
 
+    def install_native_plugin(self, path: str) -> None:
```

Comment:
Wonder how do we validate the cluster can be exit gracefully after native plugin installation. I think we saw an issue when exiting previously.